Simulator/Core/BGDriver.cpp

```diff
@@ -137,7 +137,7 @@ int main(int argc, char *argv[]) {
 
    // INPUT OBJECTS ARENT IN PROJECT YET
    // Terminate the stimulus input
-//   if (pInput != NULL)
+//   if (pInput != nullptr)
```

Comment:
File name has changed.

Reply:
Eh, you could've merged this PR first and then #170 and we would've had no problems, but I guess I'll just redo the problematic files (not sure if this is the only one of all the changed files)